test/units/testsuite-58.sh

```diff
@@ -1,71 +1,87 @@
 #!/usr/bin/env bash
 set -eux
+set -o pipefail
+
+if ! which systemd-repart &>/dev/null; then
+    echo "no systemd-repart" >/skipped
```

Comment:
To make `>/skipped` work you need to override `check_result_nspawn()` and `check_result_qemu()` with custom functions that check that file:

https://github.com/systemd/systemd/blob/4990780ed01bff58e6fa147f8e6f2cb7a568bf8f/test/TEST-55-OOMD/test.sh#L9-L20